### PR TITLE
Add description to both fct_lump and fct_lump_min

### LINKED ISSUE #195: fct_lump_min() needs better documentation
What does it actually do? Probably needs a sentence in the `@description`.

### PR DESCRIPTION
**Pull request by VincentGuyader**
Fixes #195

### CODE REVIEW

**Comment by hadley** [comment]
(FYI I changed your title and body to reflect our standard naming practices)

File: R/lump.R [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #' Lump together least/most common factor levels into "other"
-#'
+#' @description `fct_lump` lumps together least/most common factor levels into "other".
```
Comment (hadley) [inline R/lump.R]: I think it's a bit easier to read if you just document both functions here — that way you can write a single paragraph. (It is a bit more confusing when you look at the docs for `fct_lump_min()` but I think that trade off is worth it).

Also can you please use parens at the end of function names? (https://style.tidyverse.org/documentation.html#r-code)

Comment (VincentGuyader) [reply R/lump.R]: I have made changes that take into account these remarks

Regards

**Comment by hadley** [comment]
Thanks!

**Pull request merged**